aten/src/ATen/function_wrapper.py

```diff
@@ -1208,9 +1208,9 @@ def gen_namespace_function(option, multidispatch_tensors):
             # Having manual_kernel_registration for an abstract method doesn't make sense.
             assert not option['manual_kernel_registration']
         else:
-            top_env['type_method_declarations'].append(NATIVE_DISPATCH_DECLARATION.substitute(option))
-            top_env['type_method_definitions'].append(NATIVE_DISPATCH_DEFINITION_DEFAULT.substitute(option))
             if not option['manual_kernel_registration']:
+                top_env['type_method_declarations'].append(NATIVE_DISPATCH_DECLARATION.substitute(option))
```

Comment:
Do we not still want the declaration in this case? E.g. that's the pattern [here](https://github.com/pytorch/pytorch/blob/master/tools/autograd/gen_variable_type.py#L534)